packages/app/src/app/components/DelayedAnimation/index.ts

```diff
@@ -2,5 +2,5 @@ import styled from 'styled-components';
 import delayEffect from '@codesandbox/common/lib/utils/animation/delay-effect';
 
 export default styled.div`
-  ${props => delayEffect(props.delay || 0)};
+  ${({ delay }: { delay: number }) => delayEffect(delay || 0)};
```

Comment:
Yeah we can probably extract these out. I haven't made a hard decision whether we should extract out all props to interface definitions, because if we do that for all styled elements then we should also move those interfaces to `types.ts` to reduce clutter.